js/test/Ice/proxy/Client.ts

```diff
@@ -950,148 +950,147 @@ export class Client extends TestHelper {
         p2 = communicator.stringToProxy("test:opaque -e 1.1 -t 4 -v CTEyNy4wLjAuMeouAAAQJwAAAAA=");
         test(communicator.proxyToString(p2) === "test -t -e 1.1:ws -h 127.0.0.1 -p 12010 -t 10000");
 
-        if (communicator.getProperties().getPropertyAsInt("Ice.IPv6") === 0) {
```

Comment:
We don't support this property in JS

Reply:
Are all this changes related to indentation change, or did you have to fix anything else?

Reply:
Just indentation